.github/workflows/stats_merged_prs.yml

```diff
@@ -0,0 +1,24 @@
+name: Merged PR stats
+
+on:
+  pull_request:
```

Comment:
Have you tested this? Because I am pretty sure this workflow is going to fail because you cannot access secrets (e.g. `ELASTICSEARCH_KEY`) in a `pull_request` workflow.

This workflow must be a `pull_request_target` because otherwise it will be a major security risk (e.g. someone could edit `stats_merged_prs.yml` or `merged_prs.py` in a PR and gain write access to the ElasticSearch server.

Reply:
> Have you tested this? Because I am pretty sure this workflow is going to fail because you cannot access secrets (e.g. `ELASTICSEARCH_KEY`) in a `pull_request` workflow.

of course I did. I can change this to pull_request_target of course, but it works also without.

Reply:
That only worked because the PR was opened from a repo branch (i.e. not a fork branch), which is "trusted" because only people with write access to the repo can push repo branches. This will fail if a PR is opened from a fork branch.

Anyway, it should be changed to `pull_request_target` for security reasons.

Reply:
updated to use `pull_request_target`